subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -542,6 +542,44 @@ static int engine_add_observer(struct lwm2m_message *msg,
 	return 0;
 }
 
+int lwm2m_engine_update_period_min_observer(uint32_t obj_id, uint32_t res_id, uint32_t period_s)
```

Comment:
The function name is a bit difficult to understand in this order, how about `lwm2m_engine_update_observer_min_period()`?

Reply:
Way better now. Solved.